viewer/db.js

```diff
@@ -987,31 +1003,273 @@ exports.getHunt = async (id) => {
 };
 
 // Shortcut DB interactions
+// since 3.0 shortcuts are synced across clusters if user's elasticsearch is configured
+// determine if shorcuts have been synced from the local db to the remote db (remote db = user's db)
+// if there are shortcuts to sync, adds them to the remote db and deduplicates
+// shortcut names by appending the cluster or index name to the shortcut name
+async function initialShortcutsSyncToRemote () {
+  if (internals.multiES) { return; } // don't sync shortcuts for multies
+
+  const { body: doc } = await internals.usersClient7.indices.getMapping({
+    index: internals.remoteShortcutsIndex
+  });
+
+  let initSync;
+  // eslint-disable-next-line no-unreachable-loop
+  for (const index in doc) {
```

Comment:
The millennial way might be Object.keys(doc)[0]

Reply:
lol, what are you saying? That I'm OLD!? 👵

Reply:
once you have :kids:

Reply:
😭